src/Backups/BackupCoordinationStageSync.cpp

```diff
@@ -891,8 +891,19 @@ bool BackupCoordinationStageSync::checkIfHostsReachStage(const Strings & hosts,
         }
 
         if (host_info.finished)
+        {
+            if (stage_to_wait == "finalizing tables")
```

Comment:
nit: why not use the `BackupCoordinationStage::FINALIZING_TABLES` constant here?